saos-batch-jobs/src/main/java/pl/edu/icm/saos/batch/jobs/CcjRemoveJobConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package pl.edu.icm.saos.batch.jobs;
+
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.launch.support.RunIdIncrementer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import pl.edu.icm.saos.importer.commoncourt.judgment.remove.CcjRemoverProcessor;
+import pl.edu.icm.saos.importer.commoncourt.judgment.remove.CcjRemoverReader;
+import pl.edu.icm.saos.importer.commoncourt.judgment.remove.CcjRemoverWriter;
+import pl.edu.icm.saos.persistence.model.Judgment;
+import pl.edu.icm.saos.persistence.model.RemovedJudgment;
+
+/**
+ * @author madryk
+ */
+@Configuration
+public class CcjRemoveJobConfiguration {
```

Comment:
I would go with one unified phrase for this job. Now we have Ccj**Remove**JobConfiguration, but Ccj**Remover**Reader or ccJudgment**RemoveObsolete**Job.
What we are doing is deleting of judgments that have been removed in the source system, so maybe Ccj**DeleteRemoved**?